packages/snippet-bot/test/snippet-bot.ts

```diff
@@ -406,6 +412,102 @@ describe('snippet-bot', () => {
 
       requests.done();
     });
+
+    it('gives warnings about removing region tag in use', async () => {
```

Comment:
I wonder if it would be useful to put another type of test as well asserting the opposite, i.e., doesn't warn if region tag isn't being removed or isn't in use, etc.

Reply:
This is tested by other assertion :)

Reply:
Maybe I can add an explicit description for that :)

Reply:
Ok, I changed the test description. @sofisl PTAL